node/subsystem/src/messages.rs

```diff
@@ -136,45 +137,57 @@ impl CandidateValidationMessage {
 	}
 }
 
-/// Events from network.
-#[derive(Debug, Clone)]
-pub enum NetworkBridgeEvent {
-	/// A peer has connected.
-	PeerConnected(PeerId, ObservedRole),
-
-	/// A peer has disconnected.
-	PeerDisconnected(PeerId),
-
-	/// Peer has sent a message.
-	PeerMessage(PeerId, Vec<u8>),
-
-	/// Peer's `View` has changed.
-	PeerViewChange(PeerId, View),
 
-	/// Our `View` has changed.
-	OurViewChange(View),
+/// Messages received by the Collator Protocol subsystem.
+#[derive(Debug)]
+pub enum CollatorProtocolMessage {
```

Comment:
Worth adding a `relay_parent` function for parity with the rest of the subsystem messages, or does it seem unlikely that such a function would actually be useful?